test/socketpair_helper.py

```diff
@@ -1,5 +1,10 @@
 import socket
 
+try:
+    _CONNECT_ERROR = (BlockingIOError, InterruptedError)
+except:
```

Comment:
Oh sorry! I thought you meant that you didn't want `_CONNECT_ERROR = (Exception,)`. I'll fix this.